demo/campaign/preBuild/src/publicGenerator/generate.ts

```diff
@@ -0,0 +1,10 @@
+import { robotsTxt } from "./robots.txt";
+import { sitemapXml } from "./sitemap.xml";
+
+const main = () => {
```

Comment:
I'd say preBuild is not necessary for the campaigns frontend, because we don't need robotsTxt and sitemap.xml

Reply:
There are a few other components as well that aren't needed. Header and so on.
Unless we later add something like a SubscribeNewsletterBlock to the "campaigns"-demo as well.

What do you think? Does it make sense to keep the files or rather remove them

Reply:
Yes, I deleted the prebuild. Kept the other files due to the possibility that it will be needed in the future. We can still delete them at a later time in the future. 
[54bfcf7](https://github.com/vivid-planet/comet-brevo-module/pull/41/commits/54bfcf779f1d7c1b2e5f671186ad11a2244adfc3)